### PR TITLE
Port items to next tagged likhithanimma1

### PR DESCRIPTION
**Pull request by likhithanimma1**
## Proposed changes

Port #2765 and #2848 to next

## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Review by zFernand0 — APPROVED** [review]
Couldn't really test the hrecall and hmigrate operations but the changes looked almost identical 😋 

Thanks for porting these over. LGTM! 😋

**Review by JillieBeanSim — COMMENTED** [review]
@likhithanimma1 are there CHANGELOG entries for these 2 items?

**Comment by JillieBeanSim** [comment]
Hey @likhithanimma1 the CHANGELOG entries seem to have gone under the latest 3.x prerelease entry location.  Could you double check the version it should be under from main and update please. Thanks

File: packages/zowe-explorer/package.nls.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
   "uss.removeSession": "Hide Profile",
   "uss.binary": "Toggle Binary",
   "uss.uploadDialog": "Upload Files...",
+  "uss.uploadDialogBinary": "Upload Files... (Binary)",
```
Comment (t1m0thyj) [inline packages/zowe-explorer/package.nls.json:58]: I think the ellipsis (`...`) is usually appended at the very end of menu items. Should we change it to:

```suggestion
  "uss.uploadDialogBinary": "Upload Files (Binary)...",
```

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @likhithanimma1! Left a comment regarding UX that also applies to v2 🙂

**Review by traeok — APPROVED** [review]
LGTM, thanks @likhithanimma1!

**Review by zFernand0 — DISMISSED** [review]
LGTM! 😋 
there's another merge conflict though 😢

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**